google/api_core/grpc_helpers.py

```diff
@@ -24,6 +25,27 @@
 import google.auth.credentials
 import google.auth.transport.grpc
 import google.auth.transport.requests
+import google.protobuf
+
+PROTOBUF_VERSION = google.protobuf.__version__
+
+# The grpcio-gcp package only has support for protobuf < 4
+if int(PROTOBUF_VERSION.split(".")[0]) < 4:
```

Comment:
Minor: guess we can just check if the first two characters are "4." no need to split and convert.

Reply:
addressed in [6225d4f](https://github.com/googleapis/python-api-core/pull/418/commits/6225d4fad49792d88b00584f8c7596c2d02c076c)